README.md

```diff
@@ -148,3 +150,10 @@ Remove all installed dependencies:
 ```sh
 $ gitman uninstall
 ```
+
+### Specifying a Default Group
+
+Using the `default_group` attribute in gitman.yml specifies which group of 
+dependencies to install if no inputs are provided to `gitman install`. If 
+if is set to a blank string, `default_group: ''`, then all sources are 
+installed.
```

Comment:
Since I would consider this an advanced feature, I'd prefer documentation of it to go in either a new [user cases](https://github.com/jacebrowning/gitman/tree/main/docs/use-cases) or [extras](https://github.com/jacebrowning/gitman/tree/main/docs/extras) page as you see fit.